src/ncp/ncp_base.cpp

```diff
@@ -1793,6 +1793,10 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CAPS>(void)
     SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_SLAAC));
 #endif
 
+#if OPENTHREAD_CONFIG_RADIO_CHANNEL_MASK
+    SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_RADIO_CHANNEL_MASK));
```

Comment:
Is it possible to avoid introducing this new capability? If this feature is not supported, we have simply return not found error when getting or setting `SPINEL_PROP_PHY_CHAN_PREFERRED`. I'm not in favor of resulting in hundreds of caps.

Reply:
Agree that we can get rid of this new cap if we can accept the not found error, wpantund can set the preferred channel mask to 0 when not found.

@abtink, May I know your opinion?